config/sync/block_content.type.event_search.yml

```diff
@@ -0,0 +1,8 @@
+uuid: c665d0ae-6786-4663-aa28-600b929de864
```

Comment:
The goal of this task is only to create the regions, not to place blocks inside of them. It looks like this commit places a content block in the region. Please remove these blocks, export your site's config, and commit the removal of the config files which are impacted. If we merge this in as-is, it would result in "block not found" messages on prod.

Reply:
Will do, I had the blocks in there while I was adding the new region to make sure it worked as expected.  I'll remove and re-commit. Thanks!